include/ck/tensor_operation/gpu/block/blockwise_softmax.hpp

```diff
@@ -88,9 +88,8 @@ struct BlockwiseSoftmax
     __host__ __device__ void Run(CThreadBuffer& in_thread_buf, WorkspaceBuffer& reduce_work_buf)
     {
```

Comment:
recommand rewrite Run function: call CalculateRowMax fun +  CalculateRowExpSum fun

Reply:
Don't want to destroy the old `Run()`, which could be used by other codes.  Also I want to keep the output update logic visible in our kernel